src/main/java/Duke.java

```diff
@@ -25,10 +25,32 @@ public static void main(String[] args) {
             } else if (userCommand.contains("done")) {
                 int id = Integer.parseInt(userCommand.substring(5));
                 taskDone(tasks[id - 1]);
-            } else {
+            } else if (userCommand.startsWith("todo")) {
+                int contentStart = 5;
```

Comment:
probably can identify what the 5 means... it is like a magic number. Making it a constant or add comments may make it clearer